rules/opentitan.bzl

```diff
@@ -145,10 +157,13 @@ elf_to_disassembly = rule(
     incompatible_use_toolchain_transition = True,
 )
 
-def _elf_to_scrambled(ctx):
+def _elf_to_scrambled_rom_impl(ctx):
     outputs = []
     for src in ctx.files.srcs:
-        scrambled = ctx.actions.declare_file("{}.scr.40.vmem".format(src.basename))
+        scrambled = ctx.actions.declare_file("{}.scr.39.vmem".format(
+            # Remove ".elf" from file basename.
+            src.basename[:-4],
+        ))
```

Comment:
If you don't want to assume the file ends in `.elf`, you could use `replace` here:

```
"{}.scr.39.vmem".format(src.basename.replace("."+src.extension, ""))
```

Reply:
Ah thanks, still have to convince myself I can use _parts of_ Python hehe ... Fixed.